articles/virtual-machines/extensions/azure-disk-enc-windows.md

```diff
@@ -132,39 +137,44 @@ Using `AADClientCertificate`:
 
 | Name | Value / Example | Data Type |
 | ---- | ---- | ---- |
-| apiVersion | 2015-06-15 | date |
+| apiVersion | 2019-07-01 | date |
 | publisher | Microsoft.Azure.Security | string |
-| type | AzureDiskEncryptionForLinux | string |
-| typeHandlerVersion | 1.1, 2.2 | string |
+| type | AzureDiskEncryption | string |
+| typeHandlerVersion | 2.2, 1.1 | string |
 | (1.1 schema) AADClientID | xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx | guid | 
 | (1.1 schema) AADClientSecret | password | string |
 | (1.1 schema) AADClientCertificate | thumbprint | string |
-| DiskFormatQuery | {"dev_path":"","name":"","file_system":""} | JSON dictionary |
 | EncryptionOperation | EnableEncryption, EnableEncryptionFormatAll | string | 
-| KeyEncryptionAlgorithm | 'RSA-OAEP', 'RSA-OAEP-256', 'RSA1_5' | string |
-| KeyEncryptionKeyURL | url | string |
+| (optional - default RSA-OAEP ) KeyEncryptionAlgorithm | 'RSA-OAEP', 'RSA-OAEP-256', 'RSA1_5' | string |
 | KeyVaultURL | url | string |
-| (optional) Passphrase | password | string | 
-| SequenceVersion | uniqueidentifier | string |
+| KeyVaultResourceId | url | string |
+| (optional) KeyEncryptionKeyURL | url | string |
+| (optional) KekVaultResourceId | url | string |
+| (optional) SequenceVersion | uniqueidentifier | string |
 | VolumeType | OS, Data, All | string |
 
 ## Template deployment
-For an example of template deployment, see [
-Create a new encrypted Windows VM from gallery image](https://github.com/Azure/azure-quickstart-templates/tree/master/201-encrypt-create-new-vm-gallery-image).
 
-## Azure CLI deployment
+For an example of template deployment based on schema v2.2, see [Enable encryption on a running Windows VM](https://github.com/Azure/azure-quickstart-templates/tree/master/201-encrypt-running-windows-vm-without-aad).
```

Comment:
`see Azure QuickStart Template [201-encrypt-running-windows-vm-without-aad](https://github.com/Azure/azure-quickstart-templates/tree/master/201-encrypt-running-windows-vm-without-aad).`

Reply:
@simonesavi could we change the wording of this from " see Enable encryption on a running windows vm" to "see Azure QuickStart Template 201-encrypt..."

Reply:
@ejarvi done